pkg/operator/ceph/cluster/watcher.go

```diff
@@ -56,6 +56,8 @@ var (
 	nodesCheckedForReconcile = sets.New[string]()
 	networkFenceLabel        = "cephClusterUID"
 	errActiveClientNotFound  = stderrors.New("active client not found")
+	// Let's keep cephFS networkFencing disabled until we fix all the bug and do the testing
+	enableCephFsFencing = false
```

Comment:
If we're disabling it and there is no configuration option to enable it, let's just delete all the related code. Then in the future when we enable it again, we just need to revert this commit to get the code back.